lib/Log.js

```diff
@@ -209,9 +211,11 @@ class Log {
                 return this._helper.logStream(
                     this._deviceIds, 
                     this._messageHandler.bind(this),
```

Comment:
BWT do we really need `.bind(this)` here? I believe the JS functions are bound to the current scope by default. Aren't they?

Reply:
In general, they are. But these functions are called from imp-central-api logstream callbacks. In this case, they lose the original context and .bind(this) is really needed.

Reply:
Ah got it, yes. They are called from a different context. Thanks!